src/Mod/TechDraw/TechDrawTools/CommandMoveView.py

```diff
@@ -44,8 +44,8 @@ def GetResources(self):
         """Return a dictionary with data that will be used by the button or menu item."""
         return {'Pixmap': 'actions/TechDraw_MoveView.svg',
                 'Accel': "",
-                'MenuText': QT_TRANSLATE_NOOP("MoveView", "Move View"),
-                'ToolTip': QT_TRANSLATE_NOOP("MoveView", "Move a View to a new Page")}
+                'MenuText': QtGui.QApplication.translate("MoveView", "Move View", None),
+                'ToolTip': QtGui.QApplication.translate("MoveView", "Move a View to a new Page", None)}
```

Comment:
This change isn't correct: menu text and tooltips should use NOOP to add them to the translation database. The actual translation occurs elsewhere.